src/main/java/fr/openmc/core/features/leaderboards/LeaderboardManager.java

```diff
@@ -0,0 +1,335 @@
+import fr.openmc.core.features.leaderboards.listeners.PlayerListener;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.Statistic;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.entity.Display;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.*;
+
+public class LeaderboardManager {
+    @Getter
+    private static LeaderboardManager instance;
+    @Getter
+    private final Map<Integer, Map.Entry<String, Integer>> GithubContributorsMap = new TreeMap<>();
+    @Getter
+    private final Map<Integer, Map.Entry<String, String>> PlayerMoneyMap = new TreeMap<>();
+    @Getter
+    private final Map<Integer, Map.Entry<String, String>> VilleMoneyMap = new TreeMap<>();
+    @Getter
+    private final Map<Integer, Map.Entry<String, String>> PlayTimeMap = new TreeMap<>();
+    private final OMCPlugin plugin;
+    private final String repoOwner = "ServerOpenMC";
+    private final String repoName = "PluginV2";
+    private final File leaderBoardFile;
+    @Getter
+    private Location contributorsHologramLocation;
+    @Getter
+    private Location moneyHologramLocation;
+    @Getter
+    private Location villeMoneyHologramLocation;
+    @Getter
+    private Location playTimeHologramLocation;
+
+
+    public LeaderboardManager(OMCPlugin plugin) {
+        instance = this;
+        this.plugin = plugin;
+        this.leaderBoardFile = new File(OMCPlugin.getInstance().getDataFolder() + "/data", "leaderboards.yml");
+        loadLeaderBoardConfig();
+        CommandsManager.getHandler().register(new LeaderboardCommands());
+        plugin.getServer().getPluginManager().registerEvents(new PlayerListener(this), plugin);
+        new BukkitRunnable() {
+            private int i = 0;
+
+            @Override
+            public void run() {
+                if (i % 12 == 0)
+                    updateGithubContributorsMap(); // toutes les minutes pour ne pas être rate limitée par github
+                updatePlayerMoneyMap();
+                updateVilleMoneyMap();
+                updatePlayTimeMap();
+                updateHolograms();
+                i++;
+            }
+        }.runTaskTimerAsynchronously(plugin, 0, 20 * 5); //Toutes les 5 secondes en async sauf le updateGithubContributorsMap qui est toutes les minutes
```

Comment:
Met directement le résultat de ta multiplication ça évite de faire le calcul au cas où Java ne le fait pas
```suggestion
        }.runTaskTimerAsynchronously(plugin, 0, 1000); //Toutes les 5 secondes en async sauf le updateGithubContributorsMap qui est toutes les minutes
```